### DISCUSSION TITLE
Problem to update gts4 to version 3.17.0.2

### DISCUSSION THREAD

**Discussion by Cavalletta98** [zepp-health/zeppos-forum#24] [User Feedback]
Hello,i'm trying to update my gts4 to version 3.17.0.2 either from zepp app or on the Watch. On the Watch i get this error " Impossible to update". On the app, It Is blocked on the trasfering ,the update,to the Watch.
Can someone please help me?
Thanks

**Comment by duanxiong** [comment]
@Cavalletta98 , please submit a feedback(with App & device log info both included) in Zepp App after the failure, and tell us your user ID, then we can track it for you.

**Reply by Cavalletta98** [reply]
The problem was with italian language. I changed to english and It worked.
Thanks

**Reply by duanxiong** [reply]
It's great to hear that you got it worked! If available, you can still submit a feedback to help us resolve the issue with Italian language.

**Reply by Cavalletta98** [reply]
can you please attach a link where i can leave a feedback? thanks

**Reply by duanxiong** [reply]
You can find the entry for feedback in Zepp App
![img_v2_9871464e-2420-4d07-bde1-0c3e42e3e42g](https://user-images.githubusercontent.com/78085548/208401686-59b7add3-2883-4b87-9018-1ecbeb08d028.jpg)

**Reply by Cavalletta98** [reply]
Done,thank you

**Reply by duanxiong** [reply]
Thanks for your feedback! Can you share your user ID (in the page showed below) of Zepp App? So than we can track your feedback via the ID. 
PS , we have fixed an issue related to OTA yesterday and tested that Italian language worked fine on the test phones.
![img_v2_326c46a7-3efd-4726-b1f5-8d68d43ef6cg](https://user-images.githubusercontent.com/78085548/208563957-d1651c20-11fe-418a-87f6-63052a250d89.jpg)

**Reply by Cavalletta98** [reply]
Yes,my id Is: 3009471606

**Reply by duanxiong** [reply]
Thanks for your valuable feedback, we will analyse the log to check if there is any new bug that needs to be fixed.

**Reply by duanxiong** [reply]
@Cavalletta98 We have checked the log in your feedback and confirmed it's a known issue which was just fixed, thanks for your feedback again!

**Reply by Cavalletta98** [reply]
When It was fixed?

**Reply by duanxiong** [reply]
The day before yesterday. If any new problems, please feel free to feedback again :)
